app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Salesforce API</title>
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag 'sfapi' %>
+
+  </head>
+
+  <body>
+    <%= image_tag "logo.svg", class: 'logo' %>
+    <h1>Salesforce API</h1>
+
+    <div>
+      <%= yield %>
+    </div>
+    <div class="sticky-footer">
+      <p><small>Copyright &copy; 2021 Rice University</small></p>
```

Comment:
super small - but you can make the year dynamic with `<%= Date.today.year %>`

Reply:
Done. I meant to go back and do this. Thanks for catching it.